README.md

```diff
@@ -54,4 +56,32 @@ just export an empty GO_TOOLCHAIN:
 
 ```bash
 export GO_TOOLCHAIN=""
-```
\ No newline at end of file
+```
+
+## Release Workflow
+
+Decco uses semantic versioning. To release a new version of Decco. 
+
+1. (a) If you are releasing a **major** or **minor** version, such as v1.0.0 or 
+   v1.3.0, you create a new branch, named `release-x.y`, where `x` and `y` are 
+   the major and minor version respectively. Or,
+   (b) if you are releasing a **patch**, such as 1.3.5, you check out the existing 
+   branch with the appropriate major and minor version. For example, fo 1.3.5
+   check out `release-1.3`.
+
+2. On the release branch, create and push a commit that bumps 
+   [VERSION](./VERSION) to your desired version. Do not push this commit to the 
+   master branch.
+
+3. If needed, thoroughly test the release branch.
+
+4. Then, run the (Platform9 internal) `decco-release` TeamCity build. This will
+   effectively run the release script: [./hack/ci-release.sh](./hack/ci-release.sh).
+   In a preconfigured environment you could run `make release` instead.   
```

Comment:
This is a public repo from what I understand. I suppose we are ok with TeamCity references here?

Reply:
Yes, the repo is public, but it seemed to me useful (and safe) enough to publish the instructions in the repo itself rather than some doc in a separate system/repo that might be forgotten. I am okay with removing it though